int32 n_prob
int32 n_scan
---

